includes/currency-functions.php

```diff
@@ -76,8 +76,26 @@ function give_get_currencies_list() {
 	 * )
 	 *
 	 * @since 1.8.15
+	 * @deprecated 2.4.0 Use give_register_currency filter hook to register new currency.
```

Comment:
@JasonTheAdams `give_Register_currency` filter added in `2.4.0`, for this reason, I added deprecate version. Let me know if you want to change it to the latest.